contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -265,8 +265,11 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
         uint256 proposalId,
         address account,
         uint8 support,
-        uint256 weight
+        uint256 weight,
+        bytes memory data
     ) internal virtual override {
+        (data); // silence compiler warnings
```

Comment:
remove the parameter name?

Reply:
I like having the param name for self documentation purposes, but not opposed to removing if this style is preferred.